addons/website/views/snippets/s_numbers_showcase.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_numbers_showcase" name="Numbers Showcase">
+    <section class="s_numbers_showcase pt80 pb80 o_colored_level">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="8" style="gap: 24px;">
+                <div class="o_grid_item g-height-6 g-col-lg-5 col-lg-5 justify-content-start" style="--grid-item-padding-y: 32px; --grid-item-padding-x: 32px; grid-area: 1 / 1 / 9 / 7; z-index: 1;">
+                    <h2>
+                        <span class="display-2">Explore our<br/> key statistics</span>
+                    </h2>
+                    <p class="lead">Analyzing the numbers behind our success: an in-depth look at the key metrics driving our company's achievements.</p>
+                    <p>
+                        <a href="#" class="btn btn-primary">More details</a>
+                    </p>
+                </div>
+                <div
+                    class="o_grid_item g-height-4 g-col-lg-3 col-lg-3 justify-content-start o_cc o_cc3"
+                    style="--grid-item-padding-y: 32px; --grid-item-padding-x: 32px; grid-area: 1 / 7 / 5 / 10; z-index: 2;"
+                >
```

Comment:
I'm quite sure that rd-website would prefer if we don't split HTML content's attributes in different lines ( it's apparently ok for editor options)
```suggestion
                <div class="o_grid_item g-height-4 g-col-lg-3 col-lg-3 justify-content-start o_cc o_cc3"  style="--grid-item-padding-y: 32px; --grid-item-padding-x: 32px; grid-area: 1 / 7 / 5 / 10; z-index: 2;"
                >
```